syntax = "proto3";
package alert_rule;




import "monitor_sdk/model/monitor/alert_rule.proto";

/*
DetailAlertRule请求
*/
message DetailAlertRuleRequest {
    /*
    告警规则id
    */
    string id = 1 ;
}

/*
DetailAlertRule返回
*/
message DetailAlertRuleResponse {
    /*
    code
    */
    int32 code = 1 ;
    /*
    msg
    */
    string msg = 2 ;
    /*
    data
    */
    monitor.AlertRule data = 3 ;
}

/*
DetailAlertRuleApi返回
*/
message DetailAlertRuleResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    alert_rule.DetailAlertRuleResponse data = 4 ;
}

